code/modules/mining/ores_coins.dm

```diff
@@ -304,17 +304,29 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 /obj/item/coin
 	icon = 'icons/obj/economy.dmi'
 	name = "coin"
-	icon_state = "coin__heads"
+	icon_state = "coin_[customsprite]_[coinflip]"

```

Comment:
this means the default icon state is flipping

Reply:
why, coinflip is a var that picks between two sides of the coin
coin_[customsprite]_flip is the flipping animation

Reply:
ooh disregard that then